main.nf

```diff
@@ -191,30 +153,17 @@ log.info """=======================================================
     | \\| |       \\__, \\__/ |  \\ |___     \\`-._,-`-,
                                           `._,._,\'
 
-nf-core/mag v${manifest.pipelineVersion}"
+nf-core/mag v${workflow.manifest.version}
 ======================================================="""
 def summary = [:]
 summary['Pipeline Name']  = 'nf-core/mag'
-summary['Pipeline Version'] = manifest.pipelineVersion
+summary['Pipeline Version'] = workflow.manifest.version
 summary['Run Name']     = custom_runName ?: workflow.runName
 summary['Reads']        = params.reads
-summary['Fasta Ref']    = params.fasta
->>>>>>> TEMPLATE
 summary['Data Type']    = params.singleEnd ? 'Single-End' : 'Paired-End'
 summary['Max Memory']   = params.max_memory
 summary['Max CPUs']     = params.max_cpus
 summary['Max Time']     = params.max_time
```

Comment:
Why did you remove all of the output dir etc info? Would be nice to have these in the summary too!

Reply:
Same as for the channel, the info is there in the latest commit